src/controllers/expenseController.js

```diff
@@ -0,0 +1,94 @@
+const expensesService = require('../services/expensesService');
+const userService = require('../services/userService');
+
+const get = async (req, res) => {
+  const { userId, categories, from, to } = req.query;
+  const filteredExpenses = await expensesService.getAllExpenses(
+    userId,
+    categories,
+    from,
+    to,
+  );
+
+  res.status(200).send(filteredExpenses);
+};
+
+const getOne = async (req, res) => {
+  const { id } = req.params;
+  const expense = await expensesService.getExpenseById(id);
+
+  if (!expense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(expense);
+};
+
+const create = async (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+  const user = await userService.getUserById(userId);
+
+  if (!user) {
+    res.sendStatus(400);
```

Comment:
In the `create` function, you are sending a 400 status code if the user is not found. However, 400 Bad Request is typically used when the server could not understand the request due to invalid syntax. In this case, it would be more appropriate to send a 404 Not Found status code, as the user does not exist.